modules/testerina-core/src/main/ballerina/ballerina/mock/native.bal

```diff
@@ -1,6 +1,6 @@
 package ballerina.mock;
 
-@Description{value:"Modifies global connector instance's arguments for mocking purposes"}
-@Param{value:"mockableConnectorPathExpr: A path like syntax to identify and navigate the connector instances of a ballerina service"}
-@Param{value: "value: Mock value to set (e.g.: endpoint URL)"}
-native function setValue(string mockableConnectorPathExpr, string value);
\ No newline at end of file
+Description{value:"Modifies global connector instance's arguments for mocking purposes"}
+Param{value:"mockableConnectorPathExpr: A path like syntax to identify and navigate the connector instances of a ballerina service"}
+Param{value: "value: Mock value to set (e.g.: endpoint URL)"}
+public native function setValue(string mockableConnectorPathExpr, string value);
```

Comment:
Btw, please delete this file. It is not required.

Reply:
will do